src/main/java/com/google/opensesame/projects/ProjectData.java

```diff
@@ -158,4 +162,61 @@ public String getRepositoryId() {
 
     return Collections.unmodifiableList(mentors);
   }
+
+  private transient GHContent repositoryReadme = null;
+  private transient boolean repositoryReadmeLoaded = false;
```

Comment:
is there a reason why `repositoryReadmeLoaded` is transient?

Reply:
Yes. It's not a piece of data that I feel is relevant to the client and is mostly there to avoid having to reload the repository README from the GitHub API more than once.